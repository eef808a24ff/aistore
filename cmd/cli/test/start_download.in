ais create bucket $BUCKET
ais rm download --all // IGNORE
ais start download gs://lpr-vision/imagenet/imagenet_train-{000000..000140}.tgz ais://$BUCKET // FAIL "too many arguments"

ais start download gs://minikube/iso/minikube-v0.23.2.iso.sha256 ais://$BUCKET --desc="minikube" // IGNORE
ais wait download $(ais show download --regex="minikube" | awk 'NR==2 {print $1}')
ais show download --regex="minikube" // IGNORE
ais show download $(ais show download --regex="minikube" | awk 'NR==2 {print $1}')

ais start download "gs://minikube/iso/minikube-v0.23.{2..4}.iso.sha256" ais://$BUCKET --desc="minikube-range" // IGNORE
ais wait download $(ais show download --regex="minikube-range" | awk 'NR==2 {print $1}')
ais show download --regex="minikube" // IGNORE
ais show download $(ais show download --regex="minikube-range" | awk 'NR==2 {print $1}')

echo "[\"iso/minikube-v0.25.0.iso.sha256\", \"iso/minikube-v0.25.1.iso.sha256\"]" > /tmp/ais_test_dl_list.txt
ais start download gs://minikube ais://$BUCKET --object-list=/tmp/ais_test_dl_list.txt --desc="minikube-multi" // IGNORE
ais wait download $(ais show download --regex="minikube-multi" | awk 'NR==2 {print $1}')
ais show download --regex="minikube" // IGNORE
ais show download $(ais show download --regex="minikube-multi" | awk 'NR==2 {print $1}')
