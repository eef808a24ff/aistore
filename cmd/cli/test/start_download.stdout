"$BUCKET" bucket created
Done: 1 file downloaded, 0 errors
Done: 3 files downloaded (skipped: 1), 0 errors
Done: 2 files downloaded, 0 errors
